pytorch_lightning/trainer/trainer.py

```diff
@@ -1437,5 +1435,5 @@ def _determine_limit_batches(batches: Union[int, float]) -> Union[int, float]:
         return int(batches)
     else:
         raise MisconfigurationException(
-            f'You have passed invalid value {batches}, it has to be in (0, 1) or nature number.'
+            f'You have passed invalid value {batches}, it has to be in (0, 1) or an int.'
```

Comment:
This function name and msg should be updated so something else because it now checks for `val_check_interval` and `overfit_batches` as well. Any suggestions??

Reply:
do you mean it is difficult to write the test for the specific error message being returned?

Reply:
you can use `pytest` to catch the exception which matching a specific message...

Reply:
c'mon man, I know how to specify matching message with pytest 😅. I was just referring it from the user's perspective. The function just displays the wrong value but not the parameter that was set wrong here. Either I suggest passing the parameter name as a string to display it in the message or add this function to the Trainer class to easily refer it.